game/scripts/vscripts/components/musicplayer/index.lua

```diff
@@ -0,0 +1,8 @@
+--[[
+
+musicplayercontrol
+
+Player can start and pause a song using a panorama-button.
+
+]]
+require('components/musicplayer/musicplayercontrol')
```

Comment:
no new line at end of file. we have a `.editorconfig`, if you're using sublime text there's a plugin to read it in. It'll do things like this automatically